kuzzle-sdk.d.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Type definitions for kuzzle-sdk javascript client.
+ * 
+ * Project: Kuzzle Javascript SDK - https://github.com/kuzzleio/sdk-javascript
+ * Definitions by: Andréas 'ScreamZ' HANSS <andreas.hanss67@gmail.com>
+ * ---
+ * Github: https://github.com/ScreamZ
+ * LinkedIn: https://www.linkedin.com/in/andreas-hanss/
+ * 
+ * If you find any issues, feel free to report or make a pull request.
+ */
+declare module "kuzzle-sdk" {
+    export class Kuzzle {
+        constructor(protocol: EmbededProtocol | CustomProtocol, options: KuzzleOptions);
+
+        // Core
+        addListener(event: KuzzleEvent, handler: (event: any) => void): Kuzzle;
+        connect(): Promise<void>;
+        disconnect(): void;
+        flushQueue(): Kuzzle;
+        playQueue(): Kuzzle;
+        query(request: KuzzleRequest, options?: QueuableOpts): RawKuzzleResponse
+        startQueuing(): Kuzzle;
+        stopQueuing(): Kuzzle;
+
+        // Modules
+        bulk: Bulk
+        index: Index
+    }
+
+    type EmbededProtocol = "http" | "websocket" | "socketio"
+    type CustomProtocol = object
+
+    interface QueuableOpts {

```

Comment:
It's not only `queuable`, a better name could be `QueryOptions`

Reply:
I'll rename this, you're right but currently in reality this is only about the `queueable: true` parameter in each methods.